File: Examples/CSharp/Articles/ShowFormulasInsteadOfValues.cs
using System;
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class ShowFormulasInsteadOfValues
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Load the source workbook
            Workbook workbook = new Workbook(sourceDir + "sampleShowFormulasInsteadOfValues.xlsx");

            // Access the first worksheet
            Worksheet worksheet = workbook.Worksheets[0];

            // Show formulas of the worksheet
            worksheet.ShowFormulas = true;

            // Save the workbook
            workbook.Save(outputDir + "outputShowFormulasInsteadOfValues.xlsx");

            Console.WriteLine("ShowFormulasInsteadOfValues executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/ShowFormulasInsteadOfValues.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Aspose.Cells.Examples.Articles
    Public Class ShowFormulasInsteadOfValues
        Public Shared Sub Main()
            'ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = Aspose.Cells.Examples.Utils.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            Dim filePath As String = dataDir & "source.xlsx"

            'Load the source workbook
            Dim workbook As New Workbook(filePath)

            'Access the first worksheet
            Dim worksheet As Worksheet = workbook.Worksheets(0)

            'Show formulas of the worksheet
            worksheet.ShowFormulas = True

            'Save the workbook
            workbook.Save(dataDir & "output.xlsx")
            'ExEnd:1

        End Sub
    End Class
End Namespace